Dockerfile.dapper

```diff
@@ -91,6 +91,9 @@ RUN curl -sLf https://github.com/rancher/k3s/releases/download/${CATTLE_K3S_VERS
 
 RUN wget -O - ${!DOCKER_URL} > /usr/bin/docker && chmod +x /usr/bin/docker
 
+ENV YQ_URL=https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_${ARCH}
```

Comment:
Does this work on a mac ? I thought host arch would be darwin, but maybe since we are in docker its already linux, not sure how `DAPPER_HOST_ARCH` works. The download for mac is `yq_darwin_amd64`.

Reply:
I'm using a mac, but as you say it is probably linux already since we are using docker. I assumed we didn't need darwin because we don't use darwin for kubectl although there is one available.
https://github.com/rancher/rancher/blob/0390ff5bc542936c93969624c3644ddefb139b52/Dockerfile.dapper#L94